.github/workflows/e2e-embed-react.yml

```diff
@@ -25,7 +25,9 @@ jobs:
       - uses: ./.github/actions/cache-db
       - uses: ./.github/actions/cache-build
       - name: Run Tests
-        run: yarn test-e2e:embed-react
+        run: |
+          yarn test-e2e:embed-react
+          yarn workspace @calcom/embed-react packaged:tests
```

Comment:
Results for the new test.
https://github.com/calcom/cal.com/actions/runs/5130425921